ruby-challenges.rb

```diff
@@ -19,16 +30,68 @@
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
 
+# Create a method called num_sum and set parameter as array
+# Inside the method use the sum method on the array parameter to return the sum total of the numbers in the array
+# Call the method and pass in arguments
+def num_sum(array)
+    p array.sum
+end
 
+num_sum(nums_array1)
+num_sum(nums_array2)
```

Comment:
Oftentimes you will want to print the method invocation and not print inside the method.